signin/stylesheet1.css

```diff
@@ -0,0 +1,234 @@
+*{
+    padding: 0px;
+    margin: 0px;
+}
+body{
+    background-image: url(../signin/logo/loginb3.jpg);
+    
+}
+
+.start{
+    background-color: white;
```

Comment:
Use hex color codes rather than the names as these are more accurate. Or rather define CSS variables at the :root and use the var() function to get the colors.